rotkehlchen/chain/ethereum/modules/uniswap/utils.py

```diff
@@ -55,9 +56,10 @@ def uniswap_lp_token_balances(
         abi=ZERION_ABI,
         deployed_block=1586199170,
     )
-    if NodeName.OWN in ethereum.web3_mapping:
+    if ethereum.get_own_node_web3() is not None:
         chunks = list(get_chunks(lp_addresses, n=4000))
-        call_order = [NodeName.OWN]
+        node_info = min(ethereum.own_rpc_endpoints)
```

Comment:
Hmmm this seems flimsy as hell. Why do you do this? You for some reason assume the `min`, whatever `min` means in the context of an iterable of NodeNames will always be an own node? This is pretty bad code ...

You have to have a proper way to get the own node if there is any. In fact doesn't `get_own_node_web3` do that?

Reply:
removed this as it was a bad thing to have. The logic is different now